draft-ietf-homenet-front-end-naming-delegation.md

```diff
@@ -607,7 +614,7 @@ The HNA SHOULD drop any packets arriving on the WAN interface that are not issue
 # Public Homenet Reverse Zone {#sec-reverse}
 
 Public Homenet Reverse Zone works similarly to the Public Homenet Zone.
-The main difference is that ISP that provides the IP connectivity is likely also the owner of the corresponding reverse zone and administrating the Reverse Public Authoritative Servers.
+The main difference is that ISP that provides the IPv6 connectivity is likely also the owner of the corresponding IPv6 reverse zone and administrating the Reverse Public Authoritative Servers.
```

Comment:
Initially the draft was IPv6 focused and we changed it to IP as most of the mechanism were also applicable with IPv4. I am wondering if we should specify IPv6 here. I agree that only DHCPv6 have been defined and that it makes more sense with IPv6 than IPv4, so my comment is just for consistency.

Reply:
Without any hat... reverse zone is mainly for IPv6 though. So, it makes sense to have an IPv6-specific text

Reply:
I also agree here.